pennylane_lightning/src/benchmarks/CMakeLists.txt

```diff
@@ -53,33 +53,44 @@ target_link_libraries(lightning_benchmarks_dependency INTERFACE lightning_utils
 # Add benchmark_utils exe
```

Comment:
Do these benchmarks need to be commented out here?

Reply:
No, but I just wanted to compile what I needed.